src/ui/layout/mct-tree.vue

```diff
@@ -710,24 +710,30 @@ export default {
                 }
             });
         },
-        async aggregateSearchResults(results) {
+        async aggregateSearchResults(results, abortSignal) {
+            if (abortSignal.aborted) {
+                return;
+            }
+
             for (const result of results) {
-                const objectPath = await this.openmct.objects.getOriginalPath(result.identifier);
+                if (!abortSignal.aborted) {
+                    const objectPath = await this.openmct.objects.getOriginalPath(result.identifier);
```

Comment:
if we already returned out of this function because the abortSignal was aborted, why we we need to check for abortSignal.aborted again on line 719?